packages/shared/src/types/header.ts

```diff
@@ -53,3 +58,21 @@ export interface HeaderTargeting {
     lastOneOffContributionDate?: string;
     numArticles?: number;
 }
+
+export type HeaderTestGenerator = () => Promise<HeaderTest[]>;
```

Comment:
HeaderTestGenerator/RawHeaderVariantParams/RawHeaderTestParams can be removed

Reply:
Done. Also removed the ControlProportionSettings import as its not used.